### PR TITLE
add SC election AIP proposal calldata

### PR DESCRIPTION
**Pull request by DZGoldman**
description text lifted from https://forum.arbitrum.foundation/raw/20856

generated with https://github.com/ArbitrumFoundation/governance/pull/250:
```
yarn gen:proposalData --govChainProviderRPC https://arb1.arbitrum.io/rpc --actionChainIds 42161 --actionAddresses 0xd9a2e0e5d7509f0bf1b2d33884f8c1b4d4490879  --pathToDescription ./scripts/proposals/NomineeGovV2AIP/description.txt --writeToJsonPath ./scripts/proposals/NomineeGovV2AIP/nominee-gov-v2-aip.json
```

### CODE REVIEW

**Review by godzillaba — APPROVED** [review]
ran the script and got the same result, lgtm

File: scripts/proposals/NomineeGovV2AIP/description.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Arbitrum | 2024-01-24 18:14:59 UTC | #1
+
+## Constitutional
```
Comment (fredlacs) [inline scripts/proposals/NomineeGovV2AIP/description.txt:3]:
```suggestion
## Constitutional
```

File: scripts/proposals/NomineeGovV2AIP/description.txt [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+1. A 7-day ‘Contender Submission’ stage before the ‘Nominee Selection’ stage. Accordingly, the duration of the entire election process will increase from 42 days to 49 days.
+2. For candidates to apply for a Security Council position, they are required to provide a signed message from their wallet.
+3. The wording on the ArbitrumDAO Constitution is updated to correctly reflect how quorum for proposals (as well as the election process) is handled, which should also take into account ‘abstain’ votes.
+4. The wording on the ArbitrumDAO Constitution is updated to reflect the new format of the Security Council elections.
+
+The code representing the proposed changes is final and available for full review in: [https://github.com/ArbitrumFoundation/governance/pull/231](https://github.com/ArbitrumFoundation/governance/pull/231) and [https://github.com/ArbitrumFoundation/docs/pull/731](https://github.com/ArbitrumFoundation/docs/pull/731)
+
+The Governance Action contract was deployed and is available for review in https://arbiscan.io/address/0xd9a2e0e5d7509f0bf1b2d33884f8c1b4d4490879#code
+
+-------------------------
```
Comment (fredlacs) [inline scripts/proposals/NomineeGovV2AIP/description.txt:37]:
```suggestion
The Governance Action contract was deployed and is available for review in https://arbiscan.io/address/0xd9a2e0e5d7509f0bf1b2d33884f8c1b4d4490879#code

```

File: scripts/proposals/NomineeGovV2AIP/nominee-gov-v2-aip.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
...s\n\nAccordingly, this AIP aims to implement the following changes:\n\n1. A 7-day ‘Contender Submission’ stage before the ‘Nominee Selection’ stage. Accordingly, the duration of the entire election process will increase from 42 days to 49 days.\n2. For candidates to apply for a Security Council position, they are required to provide a signed message from their wallet.\n3. The wording on the ArbitrumDAO Constitution is updated to correctly reflect how quorum for proposals (as well as the election process) is handled, which should also take into account ‘abstain’ votes.\n4. The wording on the ArbitrumDAO Constitution is updated to reflect the new format of the Security Council elections.\n\nThe code representing the proposed changes is final and available for full review in: [https://github.com/ArbitrumFoundation/governance/pull/231](https://github.com/ArbitrumFoundation/governance/pull/231) and [https://github.com/ArbitrumFoundation/docs/pull/731](https://github.com/ArbitrumFoundation/docs/pull/731)\n\nThe Governance Action contract was deployed and is available for review in https://arbiscan.io/address/0xd9a2e0e5d7509f0bf1b2d33884f8c1b4d4490879#code\n\n-------------------------\n",
```
Comment (fredlacs) [inline scripts/proposals/NomineeGovV2AIP/nominee-gov-v2-aip.json:8]: this will get updated if we remove the heading and footer that I suggested above

**Review by fredlacs — CHANGES_REQUESTED** [review]

**Comment by DZGoldman** [comment]
forgive the force-push, I had branched off of the wrong base branch previously; should be all good now

**Review by godzillaba — APPROVED** [review]

**Review by godzillaba — APPROVED** [review]

**Pull request merged**